boxoffice/views/order.py

```diff
@@ -149,13 +150,20 @@ def order(item_collection):
                     message=invalid_quantity_error_msg.format(item=item.title)), 400)
 
     user = User.query.filter_by(email=buyer_form.email.data).first()
+    order_session_form = OrderSessionForm.from_json(request.json.get('order_session'))
     order = Order(user=user,
         organization=item_collection.organization,
         item_collection=item_collection,
         buyer_email=buyer_form.email.data,
         buyer_fullname=buyer_form.fullname.data,
         buyer_phone=buyer_form.phone.data)
 
+    order_session = OrderSession(**order_session_form.data)
+    if order_session.utm_campaign:
```

Comment:
Is there a context in which you'll get other fields but not `utm_campaign`? What does Sendgrid include in links? (Google Analytics documentation should be able to clarify this.)